### PR TITLE
calendar: Load holidays before events

### PR DESCRIPTION
**Pull request by nxdefiant**
Previously events were loaded and as result painted first and a following draw on a holiday would overwrite the event indicator

### CODE REVIEW

File: apps/calendar/calendar.js [hunk 1]
```diff
@@ -141,6 +141,7 @@ const calcDays = (month, monthMaxDayMap, dowNorm) => {
 };
 
 const drawCalendar = function(date) {
+  d1 = new Date();
```
Comment (bobrippling) [inline apps/calendar/calendar.js:144]: Is this left over from testing?
```suggestion
```

File: apps/calendar/calendar.js [hunk 2]
```diff
@@ -281,7 +282,11 @@ const showMenu = function() {
     },
     /*LANG*/"Exit": () => load(),
     /*LANG*/"Settings": () => {
-      const appSettings = eval(require('Storage').read('calendar.settings.js'));
+      const appSettings = () => eval(require('Storage').read('calendar.settings.js'))(() => {
+        loadSettings();
+        loadEvents();
+        showMenu();
+      });
       appSettings(() => {
```
Comment (bobrippling) [inline apps/calendar/calendar.js:290]: The function passed here as an argument is unused - was there a different change intended here?

**Review by bobrippling — COMMENTED** [review]
Looks good - two minor things

File: apps/calendar/calendar.js (hunk 1 above)
Comment (nxdefiant) [reply apps/calendar/calendar.js:144]: oops, yes.

**Comment by bobrippling** [comment]
Nice, thanks!

**Pull request merged**